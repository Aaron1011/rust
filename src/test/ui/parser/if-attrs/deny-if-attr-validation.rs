fn main() {

    #[cfg(FALSE)] //~ ERROR attributes are not yet allowed on `if` expressions
    if true { panic!() }

    #[cfg_attr(FALSE, allow(warnings))] //~ ERROR attributes are not yet allowed on `if` expressions
    if true { panic!() }

    #[allow(warnings)] if true {} //~ ERROR attributes are not yet allowed on `if` expressions
    if false {
    } else if true {
    }

    #[allow(warnings)] if let Some(_) = Some(true) { //~ ERROR attributes are not yet allowed
    }
}
