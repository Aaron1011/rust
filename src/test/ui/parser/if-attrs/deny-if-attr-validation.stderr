error: attributes are not yet allowed on `if` expressions
  --> $DIR/deny-if-attr-validation.rs:9:5
   |
LL |     #[allow(warnings)] if true {}
   |     ^^^^^^^^^^^^^^^^^^

error: attributes are not yet allowed on `if` expressions
  --> $DIR/deny-if-attr-validation.rs:14:5
   |
LL |     #[allow(warnings)] if let Some(_) = Some(true) {
   |     ^^^^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

